source/fundamentals/crud/read-operations/watch.txt

```diff
@@ -0,0 +1,235 @@
+.. _golang-watch-changes:
+
+=================
+Watch for Changes
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to open a change stream and monitor document changes.
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load these documents into the
+``tea.ratings`` collection with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/sort.go
+   :language: go
+   :dedent:
+   :start-after: begin insertDocs
+   :end-before: end insertDocs
+
+.. include:: /includes/fundamentals/tea-sample-data-ending.rst
+
+Watch() Method
+--------------
+
+To watch for changes, use the ``Watch()`` method on any of the following objects:
+
+- ``MongoCollection``
+- ``MongoDatabase``
+- ``MongoClient``
+
+The ``Watch()`` method returns a change stream for all data changes in the collection,
+database, or deployment. 
+
+You can pass a pipeline parameter to ``Watch()`` to match only certain change operations. You
+can also pass an opts parameter to specify options for creating the change stream. 
+
+Open a Change Stream
+~~~~~~~~~~~~~~~~~~~~
+
+To open a change stream and return all document changes, use ``Watch()`` with a Context
+parameter and an empty pipeline parameter.
+
+Example
+```````
+
+The following example opens a change stream on the ``tea.ratings`` collection and 
+outputs all changes:
+
+.. code-block:: go
+
+   coll := client.Database("tea").Collection("ratings")
+
+   // open a change stream with an empty pipeline parameter
+   changeStream, err := coll.Watch(context.TODO(), mongo.Pipeline{})
+   if err != nil {
+      panic(err)
+   }
+   defer changeStream.Close(context.TODO())
+
+   // iterate over the cursor to print the change stream events
+   for changeStream.Next(context.TODO()) {
+      fmt.Println(changeStream.Current)
+   }
+
+If you modify the ``tea.ratings`` collection in a separate shell, this code will print
+your changes. Inserting a document with a ``name`` value of "``White Peony``" and a 
+``rating`` value of ``4`` will output the following change event:
+   
+.. code-block:: go
+   :copyable: false
+
+   map[_id:map[_data:8262DFFF73000000262B022C0100296E5A1004FB2444E03F9A4C5A
+   B3C7E868E387116846645F6964006462DFFF538BC04F3816B8BCD10004] clusterTime:
+   {1658847091 38} documentKey:map[_id:ObjectID("62dfff538bc04f3816b8bcd1")] 
+   fullDocument:map[_id:ObjectID("62dfff538bc04f3816b8bcd1") 
+   rating:4 type:White Peony] ns:map[coll:ratings db:tea] operationType:insert]
+
+.. note::
+
+   The ``_data``, ``clusterTime``, and ``ObjectID`` values in the output will 
+   vary with each user.
+
+Pipeline Parameter
```

Comment:
Issue: A reader who isn't familiar with change streams won't know what the pipeline parameter represents.

Suggestion: Rename to "Modify the Change Stream Output"